tobac/feature_detection.py

```diff
@@ -458,6 +473,8 @@ def feature_detection_multithreshold(
                      minimum number of identified features
     min_distance:  float
                    minimum distance between detected features (m)
+    wavelength_filtering: tuple, optional
+            minimum and maximum wavelengths in km, if spectral filtering of input field is desired
```

Comment:
Good point. I think we only discussed `dxy` and found it should be in m, so that it is the same unit as needed for the tracking. But I agree that I probably should have changed the wavelengths to m as well for consistency. To be honest I just did not think about this. It could be a bit more intuitive for users to give wavelengths in km because for most use cases they are probably ~ 100 000 - 10 000 000 m, but I am happy to change this if you think consistency is more important!

Reply:
Good idea and agree - probably really best to have the same unit. Good that you took that up again, better to change this now than later! I require now to get the wavelengths in m as well and have changed the code where necessary. I have also added a warning as you suggested and a control that the value for minimum wavelength should be larger than `dxy`